sos/plugins/logs.py

```diff
@@ -50,83 +39,31 @@ def setup(self):
             if os.path.isfile(i):
                 self.add_copy_spec(i)
 
-        if self.get_option('all_logs'):
-            self.add_journal(boot="this", since=self.get_option("since"),
-                             allfields=True, output="verbose")
-            self.add_journal(boot="last", allfields=True, output="verbose")
-
-    def postproc(self):
-        self.do_path_regex_sub(
-            r"/etc/rsyslog*",
-            r"ActionLibdbiPassword (.*)",
-            r"ActionLibdbiPassword [********]"
-        )
-        self.do_path_regex_sub(
-            r"/etc/rsyslog*",
-            r"pwd=.*",
-            r"pwd=[******]"
-        )
-
-
-class RedHatLogs(Logs, RedHatPlugin):
-
-    option_list = [
-        ("log_days", "the number of days logs to collect", "", 3)
-    ]
-
-    def setup(self):
-        super(RedHatLogs, self).setup()
-        # NOTE: for historical reasons the 'messages' and 'secure' log
-        # paths for the RedHatLogs plugin do not obey the normal --all-logs
-        # convention. The rotated copies are collected unconditionally
-        # due to their frequent importance in diagnosing problems.
-        messages = "/var/log/messages"
-        secure = "/var/log/secure"
-
-        have_messages = os.path.exists(messages)
-
-        self.add_copy_spec(secure + "*")
-        self.add_copy_spec(messages + "*")
-
-        # collect three days worth of logs by default if the system is
-        # configured to use the journal and not /var/log/messages
-        if not have_messages and self.is_installed("systemd"):
-            try:
-                days = int(self.get_option("log_days"))
-            except ValueError:
-                days = 3
-            if self.get_option("all_logs") and self.get_options("since"):
-                since = self.get_options("since")
-            else:
-                since = "-%ddays" % days
-            self.add_journal(since=since)
-
-
-class DebianLogs(Logs, DebianPlugin, UbuntuPlugin):
-
-    option_list = [
-        ("log_days", "the number of days logs to collect", "", 7)
-    ]
-
-    def setup(self):
-        super(DebianLogs, self).setup()
-        days = int(self.get_option("log_days"))
-        journal = os.path.exists("/var/log/journal/")
         self.add_copy_spec([
-            "/var/log/udev",
-            "/var/log/dist-upgrade",
```

Comment:
These two won't be further collected - is that intentional?

Reply:
theoretically they should be moved elsewhere, but I shouldn't just remove them - so adding back.